packages/jest-core/src/TestScheduler.ts

```diff
@@ -171,7 +173,10 @@ export default class TestScheduler {
     const testRunners = Object.create(null);
     contexts.forEach(({config}) => {
       if (!testRunners[config.runner]) {
-        const Runner: typeof TestRunner = require(config.runner);
+        const transformer = new ScriptTransformer(config);
+        const Runner: typeof TestRunner = interopRequireDefault(
```

Comment:
The pnp test doesn't seem to work when I use `interopRequireDefault`

Reply:
I remove the transform, just tried with `interopRequireDefault` instead of a `require`